cypress/support/fragments/inventory/inventoryInstance.js

```diff
@@ -101,6 +120,43 @@ export default {
   goToHoldingView: () => {
     cy.do(viewHoldingsButton.click());
     HoldingsRecordView.waitLoading();
+  },
+
+  openHoldings: () => {
+    cy.do([
+      firstHolding.clickHeader(),
+      secondHolding.clickHeader()
+    ]);
+  },
+
+  verifySuccessCalloutMovingMessage(count) {
```

Comment:
I've already created method for verifying callout messages (see fragments/finance/FinanceHelper/checkCalloutMessage(text, calloutType)). I think we can move it to some general helper class and use one verification method in all tests. Maybe not in this PR, in scope of refactoring for example.

Reply:
I agree about future, but now I need a function with hardcoded string about moving items between holdings

Reply:
I changed my code and I decided to reuse your function, thx)